src/components/Wheel/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { useRef, useEffect, useState } from 'react';
 
 import ModalWindow from '../ModalWindow';
-import fortune_data from '../../data/fortune.json';
+import { fortune } from '../../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file